docs/man/man1/oc-cluster-join.1

```diff
@@ -0,0 +1,190 @@
+.TH "OC CLUSTER" "1" " Openshift CLI User Manuals" "Openshift" "June 2016"  ""
+
+
+.SH NAME
+.PP
+oc cluster join \- Join an existing OpenShift cluster
+
+
+.SH SYNOPSIS
+.PP
+\fBoc cluster join\fP [OPTIONS]
+
+
+.SH DESCRIPTION
+.PP
+Add a new node to an existing OpenShift cluster
+
+.PP
+Uses an existing connection to a Docker daemon to start an OpenShift node. You must provide a secret to connect to the master. Before running command, ensure that you can execute docker commands successfully (ie. 'docker ps').
```

Comment:
Do you expect that the secret might one day also allow you to use the discovery bits likely landing in 1.6? We would need to plan for how you find an API server if so.